src/Microsoft.AspNetCore.Mvc.Razor/RazorView.cs

```diff
@@ -169,6 +169,11 @@ public class RazorView : IView
 
             _diagnosticSource.BeforeViewPage(page, context);
 
+            if (page is IRazorPageExecutionFilter filter)
```

Comment:
Some way to run code before `ExecuteAsync`. Note, the interfaces \ attributes are all internal for now